Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM adoptopenjdk/openjdk11:alpine-jre
```

Comment:
Since project is using java 8 so set base image for java 8.

Reply:
Ok I will check it. Are you planning to move to Java 11?

Reply:
No, for now Java 8 🚀  is good.